File: 프로그래머스/2/12945. 피보나치 수/피보나치 수.cs
using System;
using System.Collections.Generic;
public class Solution {
    public int solution(int n) {
        List<int> fibo = new List<int>();
        fibo.Add(1);
        fibo.Add(1);
        while(fibo.Count < n)
        {
            fibo.Add((fibo[fibo.Count -2] + fibo[fibo.Count-1]) % 1234567);
        }
        return fibo[fibo.Count-1];
    }
}

File: 프로그래머스/2/12945. 피보나치 수/피보나치 수.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int n) {
    vector<int> fibo;
    fibo.push_back(1);
    fibo.push_back(1);
    while(fibo.size() < n){
        fibo.push_back((fibo[fibo.size()-1] + fibo[fibo.size() -2]) % 1234567);
    }
    return fibo[fibo.size()-1];
}